src/assets/translations/en/main.json

```diff
@@ -1443,7 +1443,9 @@
         "connectFailure": "Unable to connect Ledger wallet",
         "unknown": "An unexpected error occurred communicating with Ledger wallet",
         "appNotOpen": "%{app} not open on Ledger",
-        "appNotOpenDescription": "Please open the %{app} app on your device and try again."
+        "appNotOpenDescription": "Please open the %{app} app on your device and try again.",
+        "noDeviceConnected": "No Ledger device connected",
```

Comment:
Note @woodenfurniture - I'm unable to get to this one but only "Could not initialize device" error state, see Jams

Reply:
Correct yeah , the new flow means the lack of a device is caught by webusb as an initialization error like in the first vid below, so this case is more of a theoretical one than anything. Make more sense to have this case covered than implicitly return assuming upstream has it handled
https://github.com/shapeshift/web/pull/6986#issuecomment-2187640161